ui/src/components/htmlhintcomponents/UpdateHTMLRules.svelte

```diff
@@ -0,0 +1,104 @@
+<script>
+  import Toastr from "../misccomponents/Toastr.svelte";
+  import { CONSTS } from "../../utils/utils";
+  import Modal from "../misccomponents//Modal.svelte";
+  import LoadingFlat from "../misccomponents/LoadingFlat.svelte";
+  import slug from "slug";
+import { keys, last } from "ramda";
+  
+  export let url;
+  export let show;
+  export let loading;
+  export let user;
+
+  let saving;
+  let addedSuccess;
+
+	let selection = [];
+
+  const dismiss = () => (show = false);
+
+  const updateIgnore = async () => {
+    const selectedRules = selection.toString()
+    saving = true;
+    if (selection.length > 0) {
+      const res = await fetch(
+        `${CONSTS.API}/api/config/${user.apiKey}/htmlhintrules`,
+        {
+          method: "PUT",
+          body: JSON.stringify({
+            url,
+            selectedRules,
+          }),
+          headers: { "Content-Type": "application/json" },
+        }
+      );
+  
+      if (res.ok) {
+        saving = false;
+        show = false;
+        addedSuccess = true;
+      } else {
+        throw new Error("Failed to load");
+      }
+    } else {
+        alert('select something')
```

Comment:
Is showing alerts usually how we display validation errors to the user? 

I would change the text to be something nicer, like "Please select rule(s) to check"